src/openrct2/object/LargeSceneryEntry.h

```diff
@@ -74,6 +74,11 @@ struct LargeSceneryEntry
     uint8_t scrolling_mode;
     LargeSceneryText* text;
     uint32_t text_image;
+
+    constexpr bool HasFlag(const uint32_t _flags) const
```

Comment:
flags are uint16_t

Reply:
Ah, good catch. I'll update the enum with `: uint16_t` as well if that's okay.